scripts/spack/specs.json

```diff
@@ -13,19 +13,20 @@
     "__comment__":"#                                                                             ",
     "__comment__":"##############################################################################",
 
+    "__comment__":"# mfem disabled for intel (icpx compiler error)",
```

Comment:
I had to disable `~mfem` for intel compiler because I was getting an intel compiler bug dump report while trying to build mfem.